### DISCUSSION TITLE
raw_tex only on unknown blocks?

### DISCUSSION THREAD

**Discussion by domhal** [jgm/pandoc#9539] [Q&A]
For this .tex file

```
\begin{proposition}
Let us prove that $G$ satisfies
\begin{equation}
x^2 = G
\end{equation}
for all $x \in G$.
\end{proposition}

\begin{figure}
\begin{tikzpicture}
\draw (1,1) -- (2,3);
\end{tikzpicture}
\end{figure}
```

the native Pandoc format using `from=latex` has one `Div` containing the proposition content and one empty `Figure`. Using instead `from=latex+raw_tex` the native Pandoc format has one `RawBlock` with the proposition environment and one `Figure` containing a `RawBlock` with the tikzpicture environment.

Is there any way to get the best of both worlds? By that I mean both of the following.
- Representing LaTeX that Pandoc "understands" in the way that `from=latex` does
- Representing LaTeX that Pandoc "doesn't understand" as `from=latex+raw_tex` does

For example, I would want the above example to be represented internally as follows.

```
[ Div
    ( "" , [ "proposition" ] , [] )
    [ Para
        [ Str "Let", Space, Str "us", Space , Str "prove", Space, Str "that", Space, Math InlineMath "G", Space, Str "satisfies", SoftBreak, Math DisplayMath "x^2 = G", SoftBreak, Str "for", Space, Str "all", Space, Math InlineMath "x \\in G", Str "."]
    ],
Figure
    ( "" , [] , [] )
    (Caption Nothing [])
    [ RawBlock (Format "latex") "\\begin{tikzpicture}\n\\draw (1,1) -- (2,3);\n\\end{tikzpicture}" ]
]
```
In other words, can LaTeX files be read in such a way that the parts Pandoc doesn't know how to parse (and no other parts) are kept as raw blocks instead of being discarded?

**Comment by jgm** [comment]
No, sorry, there isn't a way to do this.
Pandoc assumes that if raw tex is okay, it's okay anywhere, and so it uses the more accurate raw tex version rather than downshifting to a Div.
